ja/lessons/basics/date-time.md

```diff
@@ -70,7 +70,7 @@ true
 ## NaiveDateTime
 
 Elixirには日付と時間を同時に含む構造体が2種類あります。
-そのうち1つは`NaiveDateTime`です。
+最初に紹介するのは`NaiveDateTime`です。
```

Comment:
(maybe) Is not it needs space around the quotes?
```suggestion
最初に紹介するのは `NaiveDateTime` です。
```

Reply:
73, 90 lines still remained.
note: need auto-correction for this later on.

Reply:
oops I missed this 🙈